.github/workflows/build.yml

```diff
@@ -25,18 +25,21 @@ jobs:
       image: ${{ matrix.config.image }}
       options: --privileged
     steps:
+    - name: Install dependencies
+      run: |
+        export DEBIAN_FRONTEND="noninteractive"
+        apt-get update
+        apt-get install -y sudo libarchive-tools curl zsync squashfs-tools aria2 desktop-file-utils wget fuse binutils file imagemagick gcc git
+        git config --global --add safe.directory "$GITHUB_WORKSPACE"
```

Comment:
Is this needed?

Reply:
Yes, this is essential to have the version.
See https://github.com/actions/checkout/issues/363#issuecomment-1915075699

Reply:
Maybe it's THIS that I was missing in my earlier attempts then. Thanks!